window.go

```diff
@@ -76,7 +76,8 @@ type Window interface {
 	// This will not destroy the window or cause the app to exit.
 	Hide()
 	// Close the window.
-	// If it is the only open window, or the "master" window the app will Quit.
+	// If it is the only open window and no menu is set via [desktop.App]
+	// SetSystemTrayMenu, or if it is the "master" window the app will Quit.
```

Comment:
I think even if it is the "master" window, as long as there is a system tray menu set, it won't quit. I feel the sentence here is a bit confusing to me.

Reply:
Are you sure @Bluebugs? I think closing a window marked as master will still exit the app.